### PR TITLE
OSDOCS-4058

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-4058](https://issues.redhat.com//browse/OSDOCS-4058)

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4058

Link to docs preview:
Page 1: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-sts-about-iam-resources.html
Page 2: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-quickstart-guide-ui.html
Page 3: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs.html
Page 4: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-ocm-role.html
Page 5: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html
Page 6: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html
Page 7: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html
Page 8: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-interactive-mode-reference.html
Page 9: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.html
Page 10: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_cli/rosa-manage-objects-cli.html
Page 11: https://58167--docspreview.netlify.app/openshift-rosa/latest/ocm/ocm-overview.html
Page 12: https://58167--docspreview.netlify.app/openshift-rosa/latest/upgrading/rosa-upgrading.html
Page 13: https://58167--docspreview.netlify.app/openshift-rosa/latest/adding_service_cluster/adding-service.html
Page 14: https://58167--docspreview.netlify.app/openshift-rosa/latest/networking/configuring-cluster-wide-proxy.html
Page 15: https://58167--docspreview.netlify.app/openshift-rosa/latest/rosa_support/rosa-troubleshooting-iam-resources.html 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: rosa_release_notes/rosa-release-notes.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ The following list describes the key features of ROSA:
 
 * **Upgrade handling**: The commands `rosa upgrade cluster` and `rosa upgrade account` roles are seamless and backward compatible with all the older ROSA versions and older {product-title} clusters. When you run these commands, the old {product-title} clusters and their relevant STS roles and policies are updated to the latest versions.
 
-* **New AWS roles supporting the ROSA UI**: The tools to manage the `ocm-role` and `user-role` and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The `ocm-role` and `user-role` are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
+* **New AWS roles supporting the ROSA UI**: The tools to manage the OpenShift Cluster Manager role and user role and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The OpenShift Cluster Manager role and user roles are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
```
Comment (aldiazRH) [inline rosa_release_notes/rosa-release-notes.adoc:36]: You can revert this change, I can make it in the what's new PR as to not create a conflict.

File: rosa_release_notes/rosa-release-notes.adoc [hunk 2]
```diff
@@ -96,7 +96,7 @@ ROSA non-STS deployment mode is no longer the preferred method for new clusters.
 
 [id="rosa-known-issues_{context}"]
 === Known issues
-* The `ocm-role` and `user-role` that are key to the ROSA provisioning wizard might get enabled accidentally in your Red Hat organization by another user. However, this behavior does not affect the usability.
+* The OpenShift Cluster Manager role and user roles that are key to the ROSA provisioning wizard might get enabled accidentally in your Red Hat organization by another user. However, this behavior does not affect the usability.
```
Comment (aldiazRH) [inline rosa_release_notes/rosa-release-notes.adoc:99]: Same as the previous comment.

**Review by aldiazRH — COMMENTED** [review]
Just a few things here and there to touch up! Looks awesome otherwise.

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a review of this? Thanks!

**Comment by yuwang-RH** [comment]
LGTM～

**Comment by jneczypor** [comment]
Hi @arendej! Can I please get a review of this PR? Thanks!

**Pull request closed without merging**